beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -3039,6 +3056,55 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
             );
         }
 
+        // Do not write to eth1 finalization cache for blocks older than 5 epochs
+        // this helps reduce noise during sync
+        if block_delay_total < self.slot_clock.slot_duration() * 160 {
```

Comment:
Epochs aren't always 32 slots (e.g. Gnosis chain)

```suggestion
        if block_delay_total < self.slot_clock.slot_duration() * 5 * T::EthSpec::slots_per_epoch() {
```